src/NzbDrone.Core/Datastore/Migration/003_add_medium_support.cs

```diff
@@ -0,0 +1,23 @@
+using FluentMigrator;
+using NzbDrone.Core.Datastore.Migration.Framework;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(3)]
+    public class add_medium_support : NzbDroneMigrationBase
+    {
+        protected override void MainDbUpgrade()
+        {
+            Alter.Table("Albums").AddColumn("Media").AsString().WithDefaultValue("");
+            Alter.Table("Tracks").AddColumn("MediumNumber").AsInt32().WithDefaultValue(0);
+            Alter.Table("Tracks").AddColumn("AbsoluteTrackNumber").AsInt32().WithDefaultValue(0);
+            Alter.Table("Tracks").AddColumn("RelativeTrackNumber").AsInt32().WithDefaultValue(0);
```

Comment:
Is this valid since the relative track number can be alphanumeric (A2, B1, A, etc.)? Also is `UPDATE` below affected by this since it should be going from int to string?

Reply:
The `trackNumber` field is storing the alpha numeric version, the `relativeTrackNumber` field tries to parse the numeric from the `trackNumber` field and store in integer form. This gives a side aware track number for cassettes and vinyls. Parse is done in skyhookproxy, if you want to double check me on it. Maybe we have no purpose for this?

Reply:
Ahh I had a hunch that that may want to be the case. I'm not so sure about the parsing...it looks like it won't parse one like `A1`? And it's up to the UI whether we need it or not I think. We could probably calculate it in the UI if we wanted to show that, but that's probably pretty wasteful if it needs to be done on every page load or something. I suppose someone could want to rename by it too.

Reply:
I don’t think we need for UI as we will just use string version or absolute number, only thing I was thinking was for rename. Maybe we remove and can add back if we think of a better use case.